index.html

```diff
@@ -1,111 +1,579 @@
+                target="_blank"
+                rel="noopener noreferrer"
+                >LinkedIn</a
+              >
+              <a
+                href="https://www.Instagram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Instagram</a
+              >
+            </div>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h3>Jason Smith</h3>
+            <p>Co-Founder</p>
+            <div>
+              <a
+                href="https://www.Twitter.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Twitter</a
+              >
+              <a
+                href="https://www.facebook.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Facebook</a
+              >
+              <a
+                href="https://www.Telegram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Telegram</a
+              >
+              <a
+                href="https://www.LinkedIn.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >LinkedIn</a
+              >
+              <a
+                href="https://www.Instagram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Instagram</a
+              >
+            </div>
+          </div>
+          <div>
+            <img src="#" alt="text" />
+            <h3>Jason Smith</h3>
+            <p>Co-Founder</p>
+            <div>
+              <a
+                href="https://www.Twitter.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Twitter</a
+              >
+              <a
+                href="https://www.facebook.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Facebook</a
+              >
+              <a
+                href="https://www.Telegram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Telegram</a
+              >
+              <a
+                href="https://www.LinkedIn.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >LinkedIn</a
+              >
+              <a
+                href="https://www.Instagram.com/yourpage"
+                target="_blank"
+                rel="noopener noreferrer"
+                >Instagram</a
+              >
             </div>
           </div>
         </div>
-      </template>
-      <!-- <script src="script.js"></script> --> <!-- uncomment script --> 
-</body>
+      </section>
+      <section class="faqs">
+        <h2>FAQs</h2>
+        <div class="details">
+          <details>
```

Comment:
Tag 'details' is a parent tag for tag 'summary'. You can see an example of usage [here](https://developer.mozilla.org/ru/docs/Web/HTML/Element/details)

Reply:
done